main.js

```diff
@@ -0,0 +1,39 @@
+const tweeter = Tweeter()
+const renderer = Renderer()
+let render = function () { renderer.renderPosts(tweeter.getPosts()) }
```

Comment:
Use `const` when declaring a function. Also, we don't usually use regular functions on one line. If you want to go with one line, use the arrow function, otherwise just hit enter ;)

But nice use of creating a function here to shorten your code later on :)